hw/ip/usbdev/dv/env/usbdev_env.core

```diff
@@ -36,6 +36,7 @@ filesets:
       - seq_lib/usbdev_setup_stage_vseq.sv: {is_include_file: true}
       - seq_lib/usbdev_pkt_sent_vseq.sv: {is_include_file: true}
       - seq_lib/usbdev_nak_trans_vseq.sv: {is_include_file: true}
+      - seq_lib/usbdev_endpoint_access_vseq.sv: {is_include_file: true}
```

Comment:
This sequence depends upon usbdev_in_trans_vseq  and needs to be included after it.

Reply:
Okay